doc/input-output-reference/src/input-for-output.tex

```diff
@@ -494,13 +494,13 @@ \subsubsection{Field: Name}\label{field-name-1-036}
 	Utility/Fuel Resource Types \tabularnewline
 	\midrule
 	\endfirsthead
-	
+
```

Comment:
This is probably fine but I wonder if this indicates your script was being implemented on lines you weren't intending